src/Sarif.Sarifer/GenerateTestDataCommand.cs

```diff
@@ -35,12 +35,12 @@ public GenerateTestDataCommand(IVsShell vsShell, IMenuCommandService menuCommand
         /// </summary>
         private void MenuCommandCallback(object caller, EventArgs args)
         {
-            this.SendDataToViewerAsync().FileAndForget(GetFileAndForgetEventName(SendDataToViewerFailureEventName));
+            this.SendDataToViewerAsync().FileAndForget(FileAndForget.EventName(SendDataToViewerFailureEventName));
         }
 
         private async Task SendDataToViewerAsync()
         {
-            string testDataFilePath = await CreateTestDataFileAsync().ConfigureAwait(continueOnCapturedContext: false);
```

Comment:
>false [](start = 112, length = 5)

Was a crashing bug: continuing on the wrong thread.

Reply:
This is odd as there shouldn't be anything requiring the UI thread until we show the data. What was crashing?

---
In reply to: [515399838](https://github.com/microsoft/sarif-visualstudio-extension/pull/239#discussion_r515399838) [](ancestors = 515399838)

Reply:
`SarifViewerInterop.IsViewerExtensionLoaded` requires the UI thread.

---
In reply to: [516032520](https://github.com/microsoft/sarif-visualstudio-extension/pull/239#discussion_r516032520) [](ancestors = 516032520,515399838)